tendermint/src/lite/verifier.rs

```diff
@@ -154,3 +160,69 @@ where
 
     Ok(())
 }
+
+/// Returns Ok if we can trust the header h2 based on h1 otherwise returns an Error.
+#[allow(clippy::too_many_arguments)] // TODO: fix this  ...
```

Comment:
We should group some params together below and can delete this ...
e.g. `(h1, h1_next_vals)` as `TrustedState` (already exists but should be a trait) and `(trusting_period, now)` as `ExpirationParams` (?)